scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua

```diff
@@ -2,13 +2,57 @@
 -- Area: Jugner Forest
 --   NM: Meteormauler Zhagtegg
 -----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+local ID = require("scripts/zones/Jugner_Forest/IDs")
+-----------------------------------
 local entity = {}
 
+-- TODO: Implement better pathing systems for guards to follow master
```

Comment:
Note: the guards don't always spawn with him: nobody knows why or what determines them spawning or not. The same is true for the other beastmen lair guardian NMs and their helpers

Reply:
Yeah if you see a bit below this line, I have a 50% chance for the guards to spawn with him. Since I didn't have a ton of info otherwise, I thought this would be the best implementation.

Reply:
the comment is more here for educating passers by on your pr :+1: 

(I'd rejected attempts to make them always spawn)